protocol/messenger_contacts.go

```diff
@@ -24,7 +24,29 @@ func (m *Messenger) acceptContactRequest(ctx context.Context, requestID string,
 	}
 
 	m.logger.Info("acceptContactRequest")
-	return m.addContact(ctx, contactRequest.From, "", "", "", contactRequest.ID, "", syncing, false, false)
+
+	response, err := m.addContact(ctx, contactRequest.From, "", "", "", contactRequest.ID, "", syncing, false, false)
+	if err != nil {
+		return nil, err
+	}
+
+	// Force activate chat
+	chat, ok := m.allChats.Load(contactRequest.From)
+	if !ok {
+		publicKey, err := common.HexToPubkey(contactRequest.From)
+		if err != nil {
+			return nil, err
+		}
+
+		chat = OneToOneFromPublicKey(publicKey, m.getTimesource())
+	}
+
+	chat.Active = true
+	if err := m.saveChat(chat); err != nil {
+		return nil, err
+	}
+
+	return response, nil
```

Comment:
you are not adding the chat to the response

Reply:
fixed